stack/templates/ingress.yaml

```diff
@@ -23,6 +23,15 @@ metadata:
                 (.Values.annotations) 
                 (.Values.ingress.annotations)
       ) }}
+      nginx.ingress.kubernetes.io/configuration-snippet: |-
```

Comment:
Makes sense to make this entire annotation conditional. If no countries are blocked you'll have an empty annotation.